gammapy/utils/fitting/iminuit.py

```diff
@@ -76,5 +76,8 @@ def make_minuit_kwargs(parameters):
         kwargs[par.name] = par.value
         if par.frozen:
             kwargs['fix_{}'.format(par.name)] = True
+        limits = par.parmin, par.parmax
+        limits = np.where(np.isnan(limits), None, limits)
+        kwargs['limit_{}'.format(par.name)] = limits
```

Comment:
Can you also pass the `err_` options to Minuit?

In Minuit, these parameter errors are used to create the initial covariance matrix that the fitter then uses.
I.e. passing `err_` is the usual way to set the "scale" of the parameters and make the fit converge or faster.
E.g. for fluxes it's usually 1e-11 but for other parameters it can be ~ 1.

Reply:
That's what I meant by initial stepsize

Reply:
Done in https://github.com/gammapy/gammapy/commit/ee2050309b2c79c7bde86d2daf04179268ac78ac